### PR TITLE
Prepare node before joining caasp cluster

### PR DESCRIPTION
**Pull request by mateiw**
# Description

Manually prepare node for joining a caasp node. Related to https://github.com/uyuni-project/uyuni/pull/2334.

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 1]
```diff
@@ -177,13 +177,16 @@ You also need to ensure that the node you want to add is registered to {productn
 
 
 .Procedure: Adding Nodes to a Cluster
+
+The node to be joined might need manual configuration before joining the cluster. Ensure the machine is used used exclusively as a {caasp} node and that swap is turned off.
```
Comment (mbologna) [inline modules/client-configuration/pages/vhm-caasp.adoc:181]:
```suggestion
The node to be joined might need manual configuration before joining the cluster. Ensure the machine is used used exclusively as a {caasp} node and that swap is turned off. For more information, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_disabling_swap
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -177,13 +177,16 @@ You also need to ensure that the node you want to add is registered to {productn
 
 
 .Procedure: Adding Nodes to a Cluster
+
+The node to be joined might need manual configuration before joining the cluster. Ensure the machine is used used exclusively as a {caasp} node and that swap is turned off. For more information, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_disabling_swap
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc:182]:
```suggestion
Before you add a node to a cluster, you might need to do some manual configuration. 
Check that the system is used exclusively as a {caasp} node, and that swap is turned off. 
For more information, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_disabling_swap



.Procedure: Adding Nodes to a Cluster
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 3]
```diff
@@ -177,13 +177,16 @@ You also need to ensure that the node you want to add is registered to {productn
...
 . In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[Join Node].
 . Select the nodes to add from the list of available nodes.
 The list of available nodes includes only nodes that are registered to {productname}, are not management nodes, and are not currently part of any cluster.
 . Follow the prompts to enter the {caasp} parameters for the nodes to be added.
 . OPTIONAL: Specify a custom ``ssh-agent`` socket that is valid only for the nodes that are being added.
 . Click btn:[Save] to schedule an action to add the nodes.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc:188]:
```suggestion
. Click btn:[Save] to schedule an action to add the nodes to the {caasp} cluster.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 4]
```diff
@@ -177,13 +177,16 @@ You also need to ensure that the node you want to add is registered to {productn
...
-During this action, {productname} prepares the nodes for joining by disabling swap, then joins the nodes to the cluster.
+During this action, {productname} joins the nodes to the {caasp} cluster.
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc:189]:
```suggestion
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 5]
```diff
@@ -177,13 +177,16 @@ You also need to ensure that the node you want to add is registered to {productn
 
 
 .Procedure: Adding Nodes to a Cluster
+
+The node to be joined might need manual configuration before joining the cluster. Ensure the machine is used used exclusively as a {caasp} node and that swap is turned off. For more information, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_disabling_swap
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:181]:
```suggestion
The node to be joined might need manual configuration before joining the cluster. Ensure the machine is used used exclusively as a {caasp} node and that swap is turned off. For more information, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_disabling_swap.
```

**Comment by Loquacity** [comment]
This one should be good to merge now, if everyone is happy.

**Pull request merged**